cookbooks/aws-parallelcluster-platform/resources/nvidia_nvlsm/nvidia_nvlsm_ubuntu22+.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Copyright:: 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+provides :nvidia_nvlsm, platform: 'ubuntu' do |node|
+  node['platform_version'].to_i >= 22
+end
+
+use 'partial/_nvidia_nvlsm_common.rb'
+use 'partial/_nvidia_nvlsm_debian.rb'
+
+def platform
```

Comment:
[Non-blocking] Nitpick: we should stop naming s3 path in a way which forces us to keep changing the URL path

Reply:
I totally agree with you. For this new dependency we applied the path structure we used so far for other dependencies.I'll keep this outside of the scope of this PR for now, but is we have capacity before the release I will fix it for NVLSM